pkg/util/write.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	"bytes"
+	"io/ioutil"
+	"os"
+)
+
+func WriteIfChanged(filename string, data []byte, perm os.FileMode) (bool, error) {
```

Comment:
set perm=0. there might actually be a bug where open syscall fails to create a new file because the permissions are zero, but fortunately we haven't run into that yet because cgroup parameter files always exist. I'll check